packages/carbon-graphs/dev/examples/controls/line.js

```diff
@@ -593,6 +593,49 @@ export const renderLinePanningWithDynamicData = (id) => {
   });
   return graph;
 };
+export const renderLinePanningWithDynamicEventline = (id) => {
+  const axisData = utils.deepClone(
+    getDemoData(`#${id}`, "LINE_TIMESERIES")
+  );
+  axisData.eventline = [
+    {
+      color: Carbon.helpers.COLORS.GREY,
+      style: {
+        strokeDashArray: "4,4"
+      },
+      value: new Date(2016, 0, 1, 8).toISOString()
+    }
+  ];
+  axisData.pan = {
+    enabled: true
+  };
+  const graphData = utils.deepClone(
+    getDemoData(`#${id}`, "LINE_TIMESERIES").data[0]
+  );
+  graphData.regions = [regions[0]];
+  const createGraph = () => {
+    graphData.eventline = [
+      {
+        color: Carbon.helpers.COLORS.BLACK,
+        style: {
+          strokeDashArray: "2,2"
+        },
+        value: new Date(2016, 0, 1, 12).toISOString()
+      }
+    ];
+    graph.reflow(graphData);
```

Comment:
try to test with different graph data instead of going with the same data, looks like things are not working from my end.

Reply:
So, as eventlines don't have any specific key to themselves and there can be multiple eventlines on the same graph. So, keeping track of the colors provided by the user would have been difficult. Hence providing them with the ability to provide different colors as well.

See description for screenshots

Reply:
In that case, everything is working as advertised. But my question is, "Why does the value of the eventline jump?"

As you show above, the initial value of the eventline is before 9:00 am, but jumps to 12:00 pm once the user begins to pan. Is this expected behavior?

Reply:
There are three ways the user can update eventlines with reflow:
1. User wants to update with new eventlines and remove the previous ones. They can also keep the old eventlines but that should be passed in the `eventline` object in panData.
2. User can keep the old eventlines by passing `eventline ` object as null or not in the panData.
3. User can completely remove eventlines just by passing an empty array in panData.